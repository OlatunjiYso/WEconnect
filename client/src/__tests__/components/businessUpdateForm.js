import React from "react";
import { shallow, mount, render } from "enzyme";

import BusinessUpdateForm from "../../components/businessUpdateForm";

describe("All tests for business form components", () => {
  const props = {
    image: {},
    business: {},
    handleChange: ()=> {},
    handleSubmit: ()=> {},
    formErrors: {},
    isFetching: false,
    handleImageChange: ()=> {}
  };

  it("should be defined", () => {
    expect(BusinessUpdateForm).toBeDefined();
  });
  it("should render the business form", () => {
    const { wrapper } = shallow(<BusinessUpdateForm {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
  it("should contain a form", () => {
    const wrapper = shallow(<BusinessUpdateForm {...props} />);
    expect(wrapper.find("form").length).toEqual(1);
  });
  it("should contain a submit button", () => {
    const wrapper = shallow(<BusinessUpdateForm {...props} />);
    const button = wrapper.find(".btn");
    expect(button.length).toEqual(1);
  });
});
