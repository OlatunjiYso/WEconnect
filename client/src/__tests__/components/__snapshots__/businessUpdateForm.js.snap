// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All tests for business form components should render the business form 1`] = `undefined`;
